tests/rbd/rbd_trash_remove.py

```diff
@@ -29,30 +59,18 @@ def run(**kw):
     2. generate IO in images
     3. Move images to trash
     4. Remove/Delete image from trash and verify that images is deleted from trash
+    5. Repeat the above steps for EC-pool.
     """
     log.info("Running Trash image remove test")
-    rbd = Rbd(**kw)
-    pool = rbd.random_string()
-    image = rbd.random_string()
-    size = "10G"
-    client = kw["ceph_cluster"].get_nodes(role="client")[0]
-    try:
-        # create rep pool and execute the test work flow
-        rbd.initial_rbd_config(rbd, pool, image, size=size)
-        client = kw["ceph_cluster"].get_nodes(role="client")[0]
-        run_fio(image_name=image, pool_name=pool, client_node=client)
-        rbd.move_image_trash(pool, image)
-        image_id = rbd.get_image_id(pool, image)
-        log.info(f"image id is {image_id}")
-        rbd.remove_image_trash(pool, image_id)
-        if rbd.trash_exist(pool, image):
-            raise ImageFoundError(" Image is found in the Trash")
-        else:
-            return 0
-
-    except RbdBaseException as error:
-        print(error.message)
-        return 1
-
-    finally:
-        rbd.clean_up(pools=[pool])
+    if kw.get("config"):
```

Comment:
Can you please rebase from master, and remove these lines 65-68. Vasi's code is merged so default values will be assigned as part of initial_rbd_config itself, and we don't need these lines.

Reply:
Done